### DISCUSSION TITLE
createArrowFiles Error

### DISCUSSION THREAD

**Discussion by YiminZhao97** [GreenleafLab/ArchR#862] [Questions / Documentation]
I am using fragment file to be input of ArchR. Attached is the link for data I used. https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM5285736.

`library(ArchR)
set.seed(1)
setwd('/Users/zhaoyimin/Desktop/COVID Project')

addArchRThreads(threads = floor(parallel::detectCores()/2)) 

inputFiles = '/Users/zhaoyimin/Desktop/COVID Project/GSM5285736_28205-0558_fragments.tsv.gz'
names(inputFiles) = 'ATAC_558'

addArchRGenome("hg19")

ArrowFiles = createArrowFiles(
  inputFiles = inputFiles,
  sampleNames = names(inputFiles),
  minTSS = 10, 
  minFrags = 10^(3.2), 
  addTileMat = TRUE,
  addGeneScoreMat = TRUE,
  force = TRUE
)`

But I always get error like this: createArrowFiles has encountered an error, checking if any ArrowFiles completed.

**Comment by rcorces** [comment]
Please upload your log file. In general, if you get an error, those can be posted in Issues. If you have a question on usage or a feature request, those belong in Discussions.

**Comment by sujong404** [comment]
l also met the same issue. 

when i tired the tutorial file in ArchR, I could get the arrow file and QualityControl directory

but, GSM5285736 files were made any arrow files   

 createArrowFiles has encountered an error, checking if any ArrowFiles completed.
(hg19)

how to solve this issue

thanks

**Reply by rcorces** [reply]
Please examine log files before posting here. Also, always include your log file if you want help. Because neither poster here included a log file, I downloaded the data myself and tried to create arrow files. This does indeed fail with the following error in the log file:
```
2022-03-09 09:23:23 : Detected 2 or less cells pass filter (Non-Zero median TSS = 1.25, median Frags = 2492) in file!
       Check inputs such as 'filterFrags' or 'filterTSS' to keep cells! Exiting!
```

It appears that, at least when analyzed with the hg19 reference genome, this data is of such low quality that essentially no cells pass filter.
